README.md

```diff
@@ -121,7 +121,16 @@ You may use the flag `--help` to view all supported features.
 
 ### Debugging
 
-The flag `--debug` enables installation of OpenSK in debug mode. You may use the GUI application J-Link RTT viewer to see the debugging messages.
+The flag `--debug` enables installation of OpenSK in debug mode. 
+
+* On nRF52840-DK board: The [Segger RTT debugging protocol](https://www.segger.com/products/debug-probes/j-link/technology/about-real-time-transfer/)
+is used. You may open the GUI application [J-Link RTT viewer](https://www.segger.com/products/debug-probes/j-link/tools/rtt-viewer/) 
```

Comment:
nit: I don't think manual instructions have to be so polite, as in "You may open". :)